tests/test_path.py

```diff
@@ -43,3 +44,12 @@ def test_is_hidden_Darwin():
     target = os.path.expanduser('~/Library')
     assert path.is_hidden(target)
     assert path.is_hidden_Darwin(target)
+
+
+def test_TreeMaker_Protocol():
+    # Ensure the validity of the TreeMaker Protocol both statically and at runtime
+    tree_maker: path.TreeMaker
+    tree_maker = pathlib.Path()
+    assert isinstance(tree_maker, path.TreeMaker)
+    tree_maker = path.Recording()
```

Comment:
No piece of code validated that `Recording` was a valid `TreeMaker` statically. As long as mypy runs on tests this is now validated.